### ISSUE TITLE
Open weekly notes open last year weekly note

### ISSUE THREAD

**Issue by TechWatching** [liamcain/obsidian-periodic-notes#150]
My weekly notes are organized by year like this:

**Comment by logista** [comment]
I also had this issue. Mine are organized/filed as `/YYYY/gggg[W]WW` and clicking on Week 45 in Calendar will open 2021W45. This has been going on since at least mid-August.

Just now I tried changing the `gggg` to `GGGG` and it works now? So that's weird. Switching back to `gggg` does break it again.

**Comment by andy47** [comment]
Same problem here. My weekly note is set to Y/gggg-MM-DD and changing the `gggg` to `GGGG` doesn't seem to make any difference.

**Comment by bennotkin** [comment]
Adding another voice to say the gggg to GGGG change doesn't fix it for me

**Comment by daves-flutterint** [comment]
I just noticed that when I try to open this weeks note, it opens last years. I have only been using this plugin for a year, so this is the first time I have noticed the problem
I have mine configured as "YYYY/MM-DD" and the folder as "Weekly"

**Comment by schobernoise** [comment]
This issue is a recurring one: #56 

I have the same issue, however in the settings menu it looks normal:
![image](https://github.com/liamcain/obsidian-periodic-notes/assets/58608284/837cb00d-54c9-4d11-8233-4b22de59ee13)

**Comment by Nordson-Rudy** [comment]
Yup this is broken.  Maybe user SchoberNoise is right, as I also also use bracket text. 

![Image](https://github.com/user-attachments/assets/c67393c8-29b1-42c4-baca-d60b7d571679)

**Comment by ChallenHB** [comment]
I was able to fix this by switching `GGGG` to `YYYY`.  

In Moment, GGGG is ISO week-year, not calendar year. It is meant to be paired with ISO week tokens like WW, not calendar month/day tokens like MM-DD. So GGGG-MM-DD mixes two date systems: ISO week-year + calendar month/day. Periodic Notes/Calendar then parses weekly note names ambiguously and can resolve to a matching week from the wrong year.
